app/controllers/trips_controller.rb

```diff
@@ -33,7 +34,7 @@ def edit
 
   # POST /trips or /trips.json
   def create
-    @trip = Trip.new(trip_params)
+    @trip = Trip.new({ **trip_params, user_id: current_user.id })
```

Comment:
this trick is 😎 but there are other alternatives. for example you can use `tap` (JFYTK)

Reply:
Nice, i will check it :)